InnoClinic.AuthorizationAPI/API/Controllers/AuthController.cs

```diff
@@ -0,0 +1,31 @@
+﻿using BLL.DTO;
+using BLL.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+
+namespace API.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+public class AuthController(IAuthService authService) : ControllerBase
+{
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [HttpPost("register")]
+    public async Task<IActionResult> RegisterAsync([FromBody] RegisterDto dto, CancellationToken cancellationToken)
+    {
+        var result = await authService.RegisterAsync(dto, cancellationToken);
+        if (!result.Succeeded) return BadRequest(new {Message = Messages.UserIsNotRegistered });
+        return Ok(new { Message = Messages.UserRegisteredSuccessfully });
+    }
+
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [HttpPost("login")]
+    public async Task<IActionResult> LogInAsynс([FromBody] LogInDto dto, CancellationToken cancellationToken)
+    {
+        var result = await authService.LogInAsync(dto, cancellationToken);
+        if (!result.Succeeded) return BadRequest(new {Message = Messages.UserIsNotLoggedIn});
+        return Ok(new {Nessage = Messages.UserLoggedInSuccessfully});
```

Comment:
Nessage?)
To avoid such errors it's better to avoid creating such entites.
In case of registration, if the result is successful we can return `Ok(result)`. If it's not - `BadRequest(result.Errors)`.